akka-docs/src/main/paradox/guide/modules.md

```diff
@@ -175,6 +229,8 @@ Some of the challenges that HTTP tackles:
 * How to stream large datasets in and out of a system using HTTP.
 * How to stream live events in and out of a system using HTTP.
 
+[Akka HTTP](http://doc.akka.io/docs/akka-http/current/scala.html) is a separate module from Akka.
```

Comment:
yes good, but perhaps also link at the top here in HTTP but before text that in order to use HTTP see the latest dependency over there